string command
---
int8 ok
